changelog/13192.feature.1.rst

```diff
@@ -0,0 +1 @@
+:func:`pytest.raises` will now print a helpful string diff if matching fails and the match parameter has ``^`` and ``$`` and is otherwise escaped.
```

Comment:
TIL you can use `.1.`, `.2.` in the news fragments. :+1:

Reply:
same ^^ Figured there had to be a way and lo and behold there was~